src/gui/shellwidget/highlight.h

```diff
@@ -25,6 +25,20 @@ class HighlightAttribute {
 	{
 	}
 
+	/// Copies a existing HighlightAttribute.
+	HighlightAttribute(const HighlightAttribute &other) noexcept :
```

Comment:
We should use the default copy constructor instead. The compiler can automatically generate code like this. http://www.cplusplus.com/articles/y8hv0pDG/

Try simply deleting the code first. If that doesn't work, we may be able to use `HighlightAttribute(const HighlightAttribute&) noexcept = default`? I can never remember the exact rules for implicit CTORS...

Reply:
Actually simply deleting those lines worked. I've misunderstood when implicit constructors are defined, and also I've completely forgot about `= default`... They're removed in 7a1434d8a186f16bdf9cf906ab099e584d380430.